hypershift-operator/controllers/nodepool/manifests.go

```diff
@@ -69,11 +69,17 @@ func TunedConfigMap(namespace, name string) *corev1.ConfigMap {
 	}
 }
 
-func PerformanceProfileConfigMap(namespace, name string) *corev1.ConfigMap {
+func PerformanceProfileConfigMap(namespace, name, nodePoolName string) *corev1.ConfigMap {
+	fullName := fmt.Sprintf("%s-%s", name, nodePoolName)
```

Comment:
I think this will have to be a bit more complex - since `name` is already a k8s object identifier and can be at the length limit itself, this `fullName` degrades to `name` in those cases and we lose the suffix / disambiguation you're trying to do here. 

You might choose to do some weighted prefixing where over-length issues get resolved by taking a prefix of both parts, if you want to keep a human-readable name. If you are not, base36(sha224(inputs)) is a good outlet. See [this example](https://github.com/openshift/hypershift/blob/921f254193afd0debb365fc96960bdba8129e970/control-plane-pki-operator/certificaterevocationcontroller/certificaterevocationcontroller.go#L447-L452) for an implementation.

Reply:
> I think this will have to be a bit more complex - since name is already a k8s object identifier and can be at the length limit itself, this fullName degrades to name in those cases and we lose the suffix / disambiguation you're trying to do here.

I wouldn't over complicated it too much, but ok, no biggie to support it